ingestion/examples/sample_data/storage/containers.json

```diff
@@ -36,7 +36,7 @@
         }
       ]
     },
-    "numberOfObjects": "50",
+    "numberOfObjects": "2147483648",
```

Comment:
can we put the exact number that we have it in the ticket

Reply:
@harshach, Done works fine reindexing also works fine .. pushed in latest commit.